include/grpc++/impl/codegen/call.h

```diff
@@ -308,7 +308,11 @@ class CallOpSendMessage {
 template <class M>
 Status CallOpSendMessage::SendMessage(const M& message, WriteOptions options) {
   write_options_ = options;
-  return SerializationTraits<M>::Serialize(message, &send_buf_, &own_buf_);
+  Status result = SerializationTraits<M>::Serialize(message, &send_buf_, &own_buf_);
+  if (own_buf_) {
```

Comment:
Under which situations is own_buf_ true?

Reply:
(should we just eliminate this flag from the API?)

Reply:
+1 -- removing this option seems like the right thing.  I'm not really sure why it's here to begin with.

Reply:
I've left ```own_buf``` in there because ```SerializationTraits``` is part of the public API.

This would be an API breaking change to remove the ability to not free the byte buffer, so I'd rather defer that change.